app/src/main/res/drawable-night/illustration_onboarding_notifications.xml

```diff
@@ -0,0 +1,200 @@
+    <path
+        android:fillColor="#1A1A1A"
+        android:pathData="M185,33.6L344,33.6A7.4,7.4 0,0 1,351.4 41L351.4,79A7.4,7.4 0,0 1,344 86.4L185,86.4A7.4,7.4 0,0 1,177.6 79L177.6,41A7.4,7.4 0,0 1,185 33.6z"
+        android:strokeWidth="1.2"
+        android:strokeColor="#4D4D4D" />
+    <path
+        android:fillAlpha="0.5"
+        android:fillColor="#FF9ABF"
+        android:pathData="M193.28,51.03C193.28,50.34 193.96,49.84 194.64,50.05L206.26,53.54C206.7,53.68 207,54.08 207,54.53V64.25C207,64.94 206.31,65.44 205.63,65.23L194.02,61.74C193.58,61.61 193.28,61.2 193.28,60.75V51.03Z"
+        android:strokeAlpha="0.5" />
+    <path
+        android:fillAlpha="0.8"
+        android:fillColor="#F789B2"
+        android:pathData="M190.64,52.69C190.64,51.99 191.33,51.49 192.01,51.7L203.62,55.2C204.06,55.33 204.36,55.73 204.36,56.18V65.9C204.36,66.6 203.67,67.09 202.99,66.89L191.38,63.39C190.94,63.26 190.64,62.86 190.64,62.4V52.69Z"
+        android:strokeAlpha="0.8" />
+    <path
+        android:fillColor="#FF5B97"
+        android:pathData="M188,54.75C188,54.06 188.69,53.56 189.37,53.77L200.98,57.26C201.42,57.39 201.72,57.8 201.72,58.25V67.97C201.72,68.66 201.04,69.16 200.36,68.95L188.74,65.46C188.3,65.32 188,64.92 188,64.47V54.75Z" />
+    <group>
+        <clip-path android:pathData="M188,54.75C188,54.06 188.69,53.56 189.37,53.77L200.98,57.26C201.42,57.39 201.72,57.8 201.72,58.25V67.97C201.72,68.66 201.04,69.16 200.36,68.95L188.74,65.46C188.3,65.32 188,64.92 188,64.47V54.75Z" />
+        <path
+            android:fillColor="#F2357A"
+            android:pathData="M188,53L201.72,57.13L194.82,63.15C194.31,63.6 193.5,63.45 193.18,62.85L188,53Z" />
+    </group>
+    <path
+        android:fillColor="#00000000"
+        android:pathData="M218,51H290"
+        android:strokeWidth="3"
+        android:strokeColor="#4D4D4D"
+        android:strokeLineCap="round" />
+    <path
+        android:fillColor="#00000000"
+        android:pathData="M218,62H280"
+        android:strokeWidth="1.2"
+        android:strokeColor="#4D4D4D"
+        android:strokeLineCap="round" />
+    <path
+        android:fillColor="#00000000"
+        android:pathData="M218,68H270"
+        android:strokeWidth="1.2"
+        android:strokeColor="#4D4D4D"
+        android:strokeLineCap="round" />
+    <path
+        android:fillColor="#00000000"
```

Comment:
Some fillColor #00000000 are not needed and we usually remove them. This goes for all notification illustrations